1.8/Dockerfile

```diff
@@ -0,0 +1,40 @@
+FROM centos/s2i-base-centos7
+
+ENV NAME=golang \
+    VERSION=1.8 
+
+ENV SUMMARY="Platform for building and running Go $VERSION based applications" \
+    DESCRIPTION="Go $VERSION available as docker container is a base platform for \
+building and running various Go $VERSION applications and frameworks. \
+Go is an easy to learn, powerful, statically typed language in the C/C++ \
+tradition with garbage collection, concurrent programming support, and memory safety features."
+
+LABEL summary="$SUMMARY" \
+      description="$DESCRIPTION" \
+      io.k8s.description="$DESCRIPTION" \
+      io.k8s.display-name="Go $VERSION" \
+      io.openshift.tags="builder,golang,golang18,rh-golang18,go" \
+      com.redhat.component="go-toolset-7" \
+      name="centos/go-toolset-7-centos7" \
+      version="1" \
+      maintainer="Jakub Čajka <jcajka@redhat.com>" \
+      usage="docker run centos/golang-18-centos7"
```

Comment:
This usage does not correspond with the updated image name `centos/go-toolset-7-centos7` now.